app/src/main/res/xml/preferences.xml

```diff
@@ -228,6 +228,10 @@
                 android:defaultValue="false"
                 android:key="display-keyboard"
                 android:title="@string/keyboard_name" />
+            <fr.neamar.kiss.preference.SwitchPreference
+                android:defaultValue="false"
```

Comment:
It would probably make sense to have this be true, so it behaves more like a normal assistant which would be ready for input right away

Reply:
Agreed, let's make it true!